src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -61,8 +94,9 @@
         // shooter can ids are range 20-29
         public static final int FLYWHEEL_1 = 0, FLYWHEEL_2 = 0, TURRET = 0, HOOD = 0;
 
-        // intake can ids are range 30-39
-        public static final int INTAKE_1 = 0, INTAKE_2 = 0, INTAKE_UP = 0, INTAKE_DOWN = 0;
+        // intake can ids are rnage 30-39
+        public static final int INTAKE_1 = 0, INTAKE_2 = 0, INTAKE_UP = 0, INTAKE_DOWN = 0,
+                RIGHT_INTAKE_COLOR_SENSOR = 0, LEFT_INTAKE_COLOR_SENSOR = 0, CENTER_INTAKE_COLOR_SENSOR = 0;
```

Comment:
Delete the color sensor IDs, as they are defined below (see LEFT_INTAKE_COLORSENSOR_PORT, etc.)